docs/spire-helm.md

```diff
@@ -53,15 +63,17 @@ For this tutorial, we can deploy it all on [minikube](https://minikube.sigs.k8s.
 ```console
 minikube start --kubernetes-version=v1.20.2
 ```
+<br>
```

Comment:
Probably shouldn't add HTML tags unless really necessary, generally markdown linting rules don't like this.